pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,11 +1,11 @@
-{stdenv, fetchurl, curl, dmd, gcc, unzip}:
+{stdenv, fetchurl, curl, dmd, gcc, unzip, git}:
```

Comment:
Ok, I see! Now I see there is a problem, since it's being installed from a source package and not a git repository there is no need for `git`.

Thank you for your help!